GabijaBern/assignment1/Anagrams.java

```diff
@@ -0,0 +1,91 @@
+//Q1: Given two strings, determine if one is an anagram of the other.
+//Parts: 1)stringSensitive()
+//		 2)stringInsensitive()
+//		 3)sentences()
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Anagrams {
+
+	
+	//-----------------first part of the question------------------------------

```

Comment:
Yes, I will try to do the documentation for the second assignment, I know how to do it, but just wrote it more for "myself". So good point, I will try to improve. :)

Reply:
Optional but it would be great if you could apply this suggestion to this assignment as well :)
The point of code reviews is to improve the code before it is merged into the main repository.
On a typical academic assignment, what matter is what you submit; on a engineering task, you want to iterate until it is as good as possible (within reason) before what is merged will stay around (other people will use it and modify it).